x/claims/keeper/hooks.go

```diff
@@ -85,7 +86,9 @@ func (k Keeper) AfterDelegationModified(ctx sdk.Context, delAddr sdk.AccAddress,
 			"address", delAddr.String(),
 			"error", err.Error(),
 		)
+		return err
```

Comment:
this should return nil as we don't want to prevent delegations if there's something wrong with the Claiming process

Reply:
got it